libs/redis/files/redis.init

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=95
+STOP=10
+
+REDIS_BIN="/usr/bin/redis-server "
+REDIS_CLI="/usr/bin/redis-cli"
+REDIS_CONFIG="/etc/redis.conf"
+REDIS_PID="/var/run/redis.pid"
+REDIS_DATA="/opt/var/lib/redis"
+REDIS_SERVER="/usr/bin/redis-server"
```

Comment:
Likely we could leave out most of these variable declaration.  I keep have a feeling that many other initscripts' using of `XX_BIN` is unnecessary.

Reply:
I removed REDIS_SERVER variable and changed REDIS_DATA to /var/lib/redis